include_core/omrgcconsts.h

```diff
@@ -84,8 +85,9 @@ typedef enum MM_GCWriteBarrierType {
 typedef enum MM_GCReadBarrierType {
 	gc_modron_readbar_illegal = OMR_GC_READ_BARRIER_TYPE_ILLEGAL,
 	gc_modron_readbar_none = OMR_GC_READ_BARRIER_TYPE_NONE,
-	gc_modron_readbar_range_check = OMR_GC_READ_BARRIER_TYPE_RANGE_CHECK,
 	gc_modron_readbar_always = OMR_GC_READ_BARRIER_TYPE_ALWAYS,
+	gc_modron_readbar_range_check = OMR_GC_READ_BARRIER_TYPE_RANGE_CHECK,
+	gc_modron_readbar_concurrent_copy_forward = OMR_GC_READ_BARRIER_TYPE_CONCURRENT_COPY_FORWARD,
```

Comment:
I'd rather see a name inferred from what the barrier does and how it's activated, and not what effectively achieves from high level point of view. 

For example it could be named: _REGION_MATCH if the barrier finds the owning region that belongs to a particular subset of regions.

Reply:
@amicic how would you feel about `_REGION_CHECK`?

Reply:
yeah, anything along that line

Reply:
fixed! thanks for the review!